src/pages/CreateCirclePage/CreateCirclePage.tsx

```diff
@@ -0,0 +1,263 @@
+  },
+  title: {
+    fontSize: 40,
+    fontWeight: 700,
+    color: theme.colors.primary,
+    margin: theme.spacing(5, 2, 2),
+  },
+  bodyInner: {
+    width: '100%',
+    maxWidth: 850,
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    padding: theme.spacing(0, 4, 1),
+    [theme.breakpoints.down('sm')]: {
+      padding: theme.spacing(0, 1, 1),
+    },
+  },
+  subtitle: {
+    color: theme.colors.primary,
+    fontWeight: 400,
+    size: 20,
+    lineHeight: 1.5,
+    textAlign: 'center',
+    marginBottom: theme.spacing(3),
+  },
+  twoColumnGrid: {
+    display: 'grid',
+    width: '100%',
+    maxWidth: 800,
+    gridTemplateColumns: '1fr 1fr',
+    gridAutoRows: '1fr',
+    columnGap: theme.spacing(3),
+    rowGap: theme.spacing(3),
+    [theme.breakpoints.down('xs')]: {
+      gridTemplateColumns: '1fr',
+      gridAutoRows: '1fr',
+    },
+  },
+  sectionTitle: {
+    color: theme.colors.primary,
+    size: 20,
+    lineHeight: 1,
+    fontWeight: 400,
+    margin: theme.spacing(6, 0, 0),
+  },
+  sectionSubtitle: {
+    color: theme.colors.primary,
+    textAlign: 'center',
+    size: 10,
+    lineHeight: 1.55,
+    fontWeight: 300,
+  },
+  longform: {
+    width: '100%',
+    margin: theme.spacing(2, 0),
+    maxWidth: 800,
+    '& > *': {
+      margin: theme.spacing(0, 0, 3),
+    },
+    '& label': {
+      alignSelf: 'flex-start',
+    },
+  },
+  saveButton: {
+    margin: theme.spacing(3, 0, 5),
+  },
+  errors: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'flex-end',
+    margin: 0,
+    minHeight: 45,
+    color: theme.colors.red,
+  },
+}));
+
+export const SummonCirclePage = () => {
+  const classes = useStyles();
+  const myAddress = useMyAddress();
+  const myAdminCircles = useMyAdminCircles();
+
+  const protocols = useMemo(
+    () =>
+      uniqBy(
+        myAdminCircles.map(({ protocol }) => protocol),
+        'id'
+      ),
+    [myAdminCircles]
+  );
+
+  const { createCircle } = useApi();
+
+  if (myAddress === undefined) {
+    return <></>;
+  }
+
+  return (
+    <div className={classes.root}>
+      <h2 className={classes.title}>Create a Circle</h2>
+      <CreateCircleForm.FormController
+        source={undefined}
+        hideFieldErrors
+        submit={({
+          h_captcha_token,
+          research_what_needs,
+          research_how_use,
+          research_org_structure,
+          research_org_link,
+          research_org_ens,
+          research_contact,
+          ...params
+        }) =>
+          createCircle(
+            { address: myAddress, ...params },
+            h_captcha_token,
+            JSON.stringify({
+              address: myAddress,
+              research_what_needs,
+              research_how_use,
+              research_org_structure,
+              research_org_link,
+              research_org_ens,
+              research_contact,
+              ...params,
+            })
+          )
```

Comment:
i see you have a todo item "Add the success modal"--my suggestion would be that there's no need for a modal; if there are no errors to show, redirect to the circle admin page

Reply:
eh, I did it before reading this

Reply:
the dark background looks good